src/vehicle.h

```diff
@@ -1286,6 +1286,9 @@ class vehicle
         /** Returns roughly driving skill level at which there is no chance of fumbling. */
         float handling_difficulty() const;
 
+        // Use grid traversal to enumerate all connected vehicles.
+        static void enumerate_vehicles( std::map<vehicle *, bool> &connected_vehicles,
```

Comment:
Would be good for the comment here to explain what the `bool` in the `map` means.

Reply:
It's the `on_map` argument to `vehicle::idle()` but I had to trace through the code to figure that out.